src/main/webapp/javascript/recipe-rendering.js

```diff
@@ -57,15 +64,22 @@ createRecipeCard = (divID, recipeInfo) => {
  * needed, such as when making a new container div.
  */
 createElement = (htmlTag, object = "", tagOptions = {}) => {
-    // Tries to create an HTML element with the given tag
+    // Create an HTML element with the given tag
     let htmlElement = document.createElement(htmlTag.toLowerCase());
 
-    let htmlText = document.createTextNode(object);
-    htmlElement.appendChild(htmlText);
+    // If the object parameter is not undefined or an empty string, it 
+    // creates a textnode and adds it to the HTML element
+    if (object !== undefined || object !== "") {
```

Comment:
Should this be &&? Same for the if-statement below

Reply:
Ahh I see what I did here. For || I would have had to use ===, but for !== I have to use &&.

Reply:
Ack and Fixed